README.md

```diff
@@ -483,6 +486,20 @@ An example of show to use bulk actions is shown below.
 />
 ```
 
+```JSX
+<DynamicDataTable
+    rows={this.state.users}
+    actions={[
+        {
+            name: 'Create user',
```

Comment:
To my understanding, actions are displayed on the right hand side of ever row in the table. If I am correct in thinking this, then this example of a create user button does not make any sense and could be confusing to a user. 

This is me thinking this this actions array would not be displayed outside of the table so a better example is the one above this of deleting a use. 

Thoughts @dextermb?

Reply:
> To my understanding, actions are displayed on the right hand side of every row in the table.

Incorrect. `actions` are shown in the `th` whereas `buttons` are displayed on every row.

Reply:
Ah ok. My bad sorry.